docs/advanced/input_files/input-main.md

```diff
@@ -2610,6 +2611,14 @@ These variables are used to control DFT+U correlated parameters
 - **Description**: The screen length of Yukawa potential. If left to default, the screen length will be calculated as an average of the entire system. It's better to stick to the default setting unless there is a very good reason.
 - **Default**: Calculated on the fly.
 
+### uramping
+
+- **Type**: Real
+- **Unit**: eV
+- **Availability**: DFT+U calculations with `mixing_restart > 0`.
+- **Description**: Once `uramping` > 0.15 eV. DFT+U calculations will start SCF with U = 0 eV, namely normal LDA/PBE calculations. Once SCF restarts when `drho<mixing_restart`, U value will increase by `uramping` eV. SCF will repeat above calcuations until U values reach target defined in `hubbard_u`.
```

Comment:
I suggest to give recommendations for `mixing_restart` with `uramping>0`, maybe the choose of mixing_restart will affect the convergence.

Reply:
solved